lesson4/src/components/WelcomeModal.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import './WelcomeModal.scss';
+
+class WelcomeModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { isVisible: false };
+        this.toggleVisible = this.toggleVisible.bind(this);
+    }
+    toggleVisible(e) {
+        this.setState(prevState => ({ isVisible: !prevState.isVisible }));
+    }
+    componentDidMount() {
+        this.setState({ isVisible: false })
```

Comment:
```suggestion
        this.setState({ isVisible: true })
```
мы хотим сделать окно видимым, поэтому isVisible (отображается ли окно) должно переключаться на `true`